bazel/cc_proto_descriptor_library/repro/repro_test.cc

```diff
@@ -0,0 +1,11 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
+// Author: kmensah@google.com (Kwasi Mensah)
+
+#include <string>
+
+#include "absl/strings/substitute.h"
+#include "bazel/cc_proto_descriptor_library/repro/base_with_any.pb.h"
+#include "bazel/cc_proto_descriptor_library/repro/base_with_any_descriptor.pb.h"
+#include "bazel/cc_proto_descriptor_library/text_format_transcoder.h"
+
```

Comment:
i don't see any test targets, could you please elaborate on what the test is intended to do?  (i'm also not clear on what the name `repro` means here?)

Reply:
Oh, actually I can just remove this whole directory. This is a set of protos I was using to help reproduce the problems I was having, but it's all obsolete now. Sorry, I should have caught this.